session/bootstrap.go

```diff
@@ -2285,6 +2290,15 @@ func upgradeToVer112(s Session, ver int64) {
 	doReentrantDDL(s, CreateMDLView)
 }
 
+func upgradeToVer113(s Session, ver int64) {
+	if ver >= version113 {
+		return
+	}
+
+	mustExecute(s, "REPLACE HIGH_PRIORITY INTO %n.%n VALUES (%?, %?);", mysql.SystemDB, mysql.GlobalVariablesTable, variable.ForeignKeyChecks, variable.On)
```

Comment:
Why use replace instead of insert ignore? It'll force update the previous value.

Reply:
`insert ignore` won't take any effect since the variable_name already exist.